ras/ras_ppcutils.py

```diff
@@ -291,10 +291,22 @@ def test_ls_vdev(self):
         """
         self.log.info("===============Executing ls-vdev tool test===="
                       "===========")
+        self.is_fail = 0
         self.run_cmd("ls-vdev")
         self.run_cmd("ls-vdev -h")
         self.run_cmd("ls-vdev -V")
-        self.error_check()
+        dev_name = self.run_cmd_out("ls-vdev").split()[1]
+        flag = True
+        for line in self.run_cmd_out("lsblk").splitlines():
+            if dev_name in line:
+                if dev_name == line.split()[0]:
+                    flag = False
+                    break
+        if flag:
+            self.is_fail += 1
```

Comment:
why two different variables here?
just increment the `self.is_fail` at line 303.
`flag` variable is not required

Reply:
yes, will do that change